src/evse_man.cpp

```diff
@@ -478,9 +478,16 @@ bool EvseManager::claim(EvseClient client, int priority, EvseProperties &target)
     DBUGF("Found slot");
     if(slot->claim(client, priority, target))
     {
-      DBUGF("Claim added/updated, waking task");
-      _evaluateClaims = true;
-      MicroTask.wakeTask(this);
+      if (client != EvseClient_OpenEVSE_Divert || client != EvseClient_OpenEVSE_Shaper) {
+        // ignore shaper & divert claims as info are already published over status
+        DBUGF("Claim added/updated, waking task");
+        StaticJsonDocument<128> event;
+        event["claims_version"] = _version++;
```

Comment:
Should increment befor sending the event I think so getClaimVersion returns the same number 

```suggestion
        event["claims_version"] = ++_version;
```

Reply:
thanks, thought it was identical, I learn something